pkg/gradle/lockfile/testdata/happy.lockfile

```diff
@@ -0,0 +1,7 @@
+# This is a Gradle generated file for dependency locking.
+# Manual edits can break the build and are not advised.
+# This file is expected to be part of source control.
+cglib:cglib-nodep:2.1.2=testRuntimeClasspath,classpath
+org.springframework:spring-asm:3.1.3.RELEASE=classpath
```

Comment:
I'd also add a record from the gradle sample:
```
org.springframework:spring-beans:5.0.5.RELEASE=compileClasspath, runtimeClasspath
```

Reply:
Done